scipy/stats/_stats_py.py

```diff
@@ -10944,6 +10947,161 @@ def linregress(x, y=None, alternative='two-sided'):
                             intercept_stderr=intercept_stderr)
 
 
+def _xp_mean(x, /, *, axis=None, weights=None, keepdims=False, nan_policy='propagate',
+             dtype=None, xp=None):
+    r"""Compute the arithmetic mean along the specified axis.
+
+    Parameters
+    ----------
+    x : real floating array
+        Array containing real numbers whose mean is desired.
+    axis : int or tuple of ints, default: None
+        If an int or tuple of ints, the axis or axes of the input along which
+        to compute the statistic. The statistic of each axis-slice (e.g. row)
+        of the input will appear in a corresponding element of the output.
+        If ``None``, the input will be raveled before computing the statistic.
+    weights : real floating array, optional
+        If specified, an array of weights associated with the values in `x`;
+        otherwise ``1``. If `weights` and `x` do not have the same shape, the
+        arrays will be broadcasted before performing the calculation. See
+        Notes for details.
+    keepdims : boolean, optional
+        If this is set to True, the axes which are reduced are left
```

Comment:
```suggestion
        If this is set to ``True``, the axes which are reduced are left
```

Reply:
Well, that would be my preference. In the past, other reviewers have criticized double backticks for short literals, so I have become inconsistent as I go back and forth between that advice and my natural tendency (code should render in monospaced font, and `True` is code).

Note that I was able to get https://github.com/numpy/numpydoc/pull/525 merged last week, and I opened https://github.com/pydata/pydata-sphinx-theme/issues/1852, but I did not handle this aspect of the issue. I'll go ahead and open another issue along these lines in the `numpydoc` repo.

Reply:
sure, I'm certainly +1 for the record